src/components/Avatar/Avatar.tsx

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 
 import * as RadixAvatar from '@radix-ui/react-avatar';
+
+import { Status } from '../Status';
```

Comment:
why do we leave spacings here?

Reply:
To group the imports :) In zUI we had a pattern to follow. This is just to keep the imports clean and easily readable. I think we should do it in zOS, but obviously not really a priority!

Reply:
ok makes sense :+1: